hr_timesheet_analysis/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
@alexey-pelykh If not any tests then remove a file.

Reply:
done